src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/CaseDataController.java

```diff
@@ -211,10 +211,14 @@ private void setPaperCaseData(Map<String, Object> caseData) {
 
     private void setOrganisationPolicy(CaseDetails caseDetails) {
         if (caseDataService.isContestedApplication(caseDetails) || caseDataService.isConsentedApplication(caseDetails)) {
+
             Map<String, Object> appSolPolicy = buildOrganisationPolicy(APP_SOLICITOR_POLICY);
-            caseDetails.getData().put(ORGANISATION_POLICY_APPLICANT, appSolPolicy);
+            if (!caseDataService.isNotEmpty(ORGANISATION_POLICY_APPLICANT, caseDetails.getData())) {
```

Comment:
you are negating a negation. why not if(caseDataService.isEmpty(...)) ?

Reply:
Thanks Mike, removed Null checker as not needed any more, issue was elsewhere.